content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -28,6 +28,8 @@ By default, pods are non-isolated; they accept traffic from any source.
 
 Pods become isolated by having a NetworkPolicy that selects them. Once there is any NetworkPolicy in a namespace selecting a particular pod, that pod will reject any connections that are not allowed by any NetworkPolicy. (Other pods in the namespace that are not selected by any NetworkPolicy will continue to accept all traffic.)
 
+Network policies do not conflict, they are additive. If policies select a pod, the pod is restricted to what is allowed by the union of those policies' ingress/egress rules. Thus, order of evaluation does not affect the policy result.
```

Comment:
really nit:  "If any policy selects a pod"

Reply:
just in this context, "If policies select a pod" could almost be worded "If multiple policies select a pod" , since it's about explaining that situation, of multiple rules, rather than the case of a single policy.